server_tests/test/clientTests.js

```diff
@@ -1479,33 +1558,33 @@ describe("RUST API TEST SUITE", function() {
                 if (!stopped) {
                     testClient.bublCaptureVideo(sessionId)
                     .then( expectError,
-                        (err) => {Comparison.cameraInExclusiveUseError(err);
+                        (err) => {validate.error(err.error.response.body, schema.names.commandBublCaptureVideo, schema.errors.cameraInExclusiveUse);
```

Comment:
This stopped assignment needs to go outside of the promise (the first thing we do in the `if`).